src/main/java/uk/gov/companieshouse/company/profile/controller/CompanyProfileController.java

```diff
@@ -35,7 +35,6 @@ public CompanyProfileController(CompanyProfileService companyProfileService) {
         return companyProfileService.get(companyNumber)
                 .map(document ->
```

Comment:
Hi, I dont see where we handling default spring de-serialisation exceptions if we send invalid json in payload from client?

Reply:
Should occur during `save` in `updateInsolvencyLink`, I've also just added a small update to throw bad request when generating the update query - there's currently multiple points of failure. This whole method will be reworked as part of a ticket you've raised as well so the error handling will also evolve :)

Reply:
Clarified further - we do throw a 400 - intercepting HttpMessageNotReadableException in Controller advice!